egs/tedlium3/ASR/RESULTS.md

```diff
@@ -0,0 +1,68 @@
+## Results
+
+### TedLium3 BPE training results (Transducer)
+
+#### Conformer encoder + embedding decoder
+
+Using the codes from this PR commit https://github.com/k2-fsa/icefall/pull/183/commits/536ad2252e2d406f24a681743d98bd5f90801b97.
+
+Conformer encoder + non-current decoder. The decoder
+contains only an embedding layer and a Conv1d (with kernel size 2).
+
+The WERs are
+
+|                                    |     dev    |    test    | comment                                  |
+|------------------------------------|------------|------------|------------------------------------------|
+|          greedy search             | 7.31       | 6.73       | --epoch 29, --avg 15, --max-duration 100 |
+|      beam search (beam size 4)     | 7.12       | 6.58       | --epoch 29, --avg 15, --max-duration 100 |
+| modified beam search (beam size 4) | 7.20       | 6.65       | --epoch 29, --avg 15, --max-duration 100 |
```

Comment:
To use `modified beam search`, you have to enable
https://github.com/k2-fsa/icefall/blob/ac7c2d84bc44dd83a5653f68677ae1ef16551eea/egs/librispeech/ASR/transducer_stateless/train.py#L141-L150
in your `train.py`.